platform/android/java/lib/src/org/godotengine/godot/utils/GodotNetUtils.java

```diff
@@ -79,4 +85,29 @@ public void multicastLockRelease() {
 			Log.e("Godot", "Exception during multicast lock release: " + e);
 		}
 	}
+
+	static public String getCACertificates() {
```

Comment:
Can you add a javadoc comment and link to the documentation/reference guide for retrieving certificates on Android.